dev-support/examples/dp-tf/readme.md

```diff
@@ -0,0 +1,23 @@
+# TF Distributed Example
+
+## Usage
+
+This is an easy example of how to track the metric and paramater in submarine-sdk. Basically, the sdk will detect which experiment and worker-id are you at, and log your data to the corresponding place.
```

Comment:
This should explain the function of tf code

Reply:
@ByronHsu  Thanks, I have updated the README, please check.